fm/views/user.py

```diff
@@ -29,8 +29,9 @@
 from fm.thirdparty.spotify import (
     PlaylistSerializer,
     SpotifyApi,
-    TrackSerializer
+    TrackSerializer,
 )
+from fm.serializers.spotify import TrackSerializer as FMTrackSerializer
```

Comment:
What is difference between `thirdparty.spotify.TrackSerializer` and `serializers.spotify.TrackSerializer`? Which is actually `FMTrackSerializer`?